File: src/main/java/com/core/base/corebase/client/dto/AuthDto.java
package com.core.base.corebase.client.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

public class AuthDto {

    @Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class LoginRes {
        private String accessToken;
        private String refreshToken;
    }

    @Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class TokenRefreshReq {
        private String refreshToken;
    }

    @Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class TokenRefreshRes {
        private String accessToken;
    }
}

File: src/main/kotlin/com/core/base/corebase/client/dto/AuthDto.kt
package com.core.base.corebase.client.dto

object AuthDto {

    class LoginRes(
        val accessToken: String,
        val refreshToken: String
    )

    class TokenRefreshReq(
        val refreshToken: String
    )

    class TokenRefreshRes(
        val accessToken: String
    )

}